beautysh/beautysh.py

```diff
@@ -31,6 +48,44 @@ def write_file(self, fp, data):
         with open(fp, 'w') as f:
             f.write(data)
 
+    def detect_function_style(self, test_record):
```

Comment:
Could you add some comments here? I have some concerns about how robust this function is.

Reply:
ok, done